assets/fresh/partials/_utils.scss

```diff
@@ -87,12 +87,13 @@ body #backtotop a {
 #backtotop a:after {
     outline: none;
     content: "\f106";
-    font-family: "FontAwesome";
+    font-family: "Font Awesome\ 5 Free";
     position: relative;
     display: block;
     top: 50%;
     -webkit-transform: translateY(-55%);
     transform: translateY(-55%);
+    font-weight: 900;

```

Comment:
Why setting the weight?

Reply:
Good question: it wasn't working without it. I believe in the past Font Awesome had one font, but now that they've re-written it they have multiple fonts (I included the free ones which I'm guessing most people will use). The fonts have a particular weight that has to be set. Before I had that included some icons wouldn't render. [Here's a little bit more information on it](https://forum.axure.com/t/font-awesome-5-solid-font-weight-changing-from-900-to-400/53548/2). I'm open to changing it if there's another option?